drivers/volume/csi/csi.go

```diff
@@ -684,7 +684,8 @@ func (c *csi) cleanupSnapshotsForRestore(
 		if err != nil {
 			return err
 		}
-		for _, vs := range vsList.Items {
+		for _, val := range vsList.Items {
+			vs := val
```

Comment:
We need to fix it in c.v1SnapshotRequired. case as well.

Reply:
fixed